tests/test_servestatic.py

```diff
@@ -16,7 +16,7 @@
 import pytest
```

Comment:
Needs an additional redirect test within `test_asgi.py`

Reply:
Added tests and fixed asgi behavior in fc0db6d